src/page-content/navigation/header.json

```
@@ -25,11 +25,11 @@
       "mainMenu": [
         {
           "title": "Ory Open Source",
-          "description": "Ory builds free and open software",
+          "description": "Ory builds open software",
```

```suggestion
          "description": "Ory builds open software",
```

free as in freedom not "for free" ! 
people want to pay us, they just want the freedom to modify things

---

```suggestion
          "description": "Ory builds free and open software",
```